docs/python/coding-pack-python.md

```diff
@@ -0,0 +1,145 @@
+---
+Order:
+Area: python
+TOCTitle: Coding Pack for Python
+ContentId: 904C9E86-A36A-4FB2-8A4C-E597B02236A5
+PageTitle: Coding Pack for Python Support
+DateApproved: 8/31/2020
+MetaDescription: Installation and support information for the Coding Pack for Python.
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Coding Pack for Python
+
+The Coding Pack for Python helps you quickly set up a Python coding environment with Visual Studio Code. The standalone installer helps you install a Python interpreter, Visual Studio Code, extensions that provide support for Python in Visual Studio Code, and a number of common and useful Python packages.
+
+## What's installed by the Coding Pack for Python
+
+The Coding Pack for Python installs the key components you need to use Visual Studio Code for Python development. Specifically, it installs:
+
+- Visual Studio Code
+- The Python, Pylance, and Gather VS Code extensions
+- The Python runtime (CPython 3.8.5)
+- Useful Python packages
+  - jupyter
+  - numpy
+  - sklearn
+  - pandas
+  - Matplotlib
+
+Along with the tools and packages necessary for Python development, the Coding Pack also configures common user settings and PowerShell. This includes Python extension settings, such as the default interpreter and language server, as well as execution policies to allow for virtual environment activation in the terminal.
+
+>**Note**: If there was an existing version of Visual Studio Code installed on your machine, your settings.json will not be overwritten and you’ll need to configure Python settings yourself.
```

Comment:
Can we move this below the Getting started section? Just trying to keep the main CTA (the download link) above the fold.

Reply:
Sure thing, changed in the latest push.